src/Myra/Graphics2D/UI/TextBlock.cs

```diff
@@ -189,6 +242,12 @@ protected override Point InternalMeasure(Point availableSize)
 				result = _formattedText.Measure(_wrap ? width : default(int?));
```

Comment:
Can you make it so this measure will be called only if autoEllipsis is disabled?